application/collateral/actions.tsx

```diff
@@ -123,7 +123,7 @@ const useCollateralActions = () => {
 
       const response = await descent.borrowCurrency(amount);
 
-      listener({
+      await listener({
         hash: response?.hash,
         amount,
         type: 'borrow',
```

Comment:
why do we have an `await` here? the purpose of the listener is to carry out an action when the tx hash completes.......Do we want the action to remain pending until then?

Reply:
yeah. that's how it's supposed to be.

Reply:
then do we need to do that for all the actions in collateral?

Reply:
yeah. we should await until the listener carries out the said action. I updated that for all the actions in the collateral